query/apigateway/apigateway_rest_api_stages_configured_with_ssl_certificate.sql

```diff
@@ -0,0 +1,16 @@
+select
```

Comment:
`query/apigateway/apigateway_rest_api_stage_use_ssl_certificate.sql`

Reply:
Please note the change to singular (most if not all of our query names use singular resource names, not plural)